src/impl/process_regular.c

```diff
@@ -283,7 +284,7 @@ int process_regular(file_t *newdir, const char *newpath, const char *oldpath,
 /* Private Impl ***************************************************************/
 
 
-int copy_fd(int dirfd, const char *pathname, struct stream *stream, uid_t uid, 
+int copy_fd(int dirfd, const char *pathname, struct stream *stream, uid_t uid,
         gid_t gid)
```

Comment:
OK, I see this isn't actually a new parameter. The change is a whitespace fix.  Regardles, move to other patch.